maps-compose/src/main/java/com/google/maps/android/compose/CameraPositionState.kt

```diff
@@ -173,9 +173,14 @@ class CameraPositionState(
      * suspend until a map is bound and animation will begin.
      *
      * This method should only be called from a dispatcher bound to the map's UI thread.
+     *
+     * @param update the change that should be applied to the camera
+     * @param durationMs The duration of the animation in milliseconds. If null is provided, the
+     * default animation duration will be used. Otherwise, the value provided must be strictly
+     * positive, otherwise an [IllegalArgumentException] will be thrown.
      */
     @UiThread
-    suspend fun animate(update: CameraUpdate) {
+    suspend fun animate(update: CameraUpdate, durationMs: Int? = null) {
```

Comment:
I typically prefer values like `MIN_VALUE` or `MAX_VALUE` over `null` when overloading a variable with other semantics (like the "default" behavior here) to keep it from being nullable if nullability isn't otherwise required. Although I don't think I've seen any official best practices for this on Kotlin so other points of view are welcome.

I assume you're not overloading `animate()` to keep the implementation simpler?

Reply:
Yes, ideally a constant value would be nice for `durationMs` but by default the SDK has different durations based on the value of `update`. And yes, I went with this implementation to keep things simple :)

Reply:
Just to clarify, I was thinking of something like this to avoid the nullable `?` and use `MAX_VALUE` as the "default" flag instead of `null`:

```kotlin
    suspend fun animate(update: CameraUpdate, durationMs: Int = MAX_VALUE) {
    ...
        if (durationMs == MAX_VALUE) {
            map.animateCamera(update, cancelableCallback)
        } else {
            map.animateCamera(update, durationMs, cancelableCallback)
        }
```

Reply:
Gotcha—I do like that better. I integrated your feedback in 3f8c1d6